setup.py

```diff
@@ -75,6 +75,7 @@
         "plone.app.viewletmanager",
         "plone.app.vocabularies",
         "plone.app.workflow",
+        "plone.app.z3cform >= 4.1.0",
```

Comment:
@jensens is it ok to define dependency here? Maybe this causes more circular import troubles ...

Reply:
This is probably a problem. Need to check. A good little task for ArtSprint!

Reply:
This PR was merged, but was this conversation resolved somewhere?

Reply:
Sorry for silence ... @jensens looked at it here at ArtSprint and it didn't have any circular issues.